### PR TITLE
IA-1651: doc updates; additional tf validation in circle; mdlint added to precommit

### PR DESCRIPTION
**Pull request by mattjorg**

### CODE REVIEW

**Review by bushong1 — DISMISSED** [review]
Doc looks good.  Just a thought, should we change the default cron to be `rate(1 week)` instead of the strangely specific time?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 
+## Unreleased Changes
 
-# Released Changes
+## Released Changes
 
-## 2.0.0
+### 2.1.0
+
+* Schedule expression variable changed to a rate based expression
```
Comment (bushong1) [inline CHANGELOG.md]: Clarify the it's just the default value that has changed

**Review by bushong1 — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mattjorg) [reply CHANGELOG.md]: Fixed

**Review by bushong1 — APPROVED** [review]
inspector gadget shaka

**Pull request merged**